jovo-platforms/jovo-platform-alexa/src/core/AlexaRequest.ts

```diff
@@ -16,6 +18,7 @@ export interface Session {
 
 export interface Context {
     System: System;
+    Viewport: Viewport;
```

Comment:
Do Echo Dot requests have the Viewport property?

Reply:
dots won't send it -> made optional